numhask/src/NumHask/Algebra/Abstract/Field.hs

```diff
@@ -84,27 +76,27 @@ class (Field a) =>
   sqrt :: a -> a
   sqrt a = a ** (one / (one + one))
```

Comment:
(Note to self) We really need NoImplicitPrelude etc. to work around the constraints introduced by numerical constants